CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
 ## MASTER
+## Dotenv 0.12.0.0
```

Comment:
@CristhianMotoche I believe this is beyond the scope of the discussion, but I'd like to add a random thought here:

We previously discussed `semantic-release` which does not support PVP versioning per se, however, I believe we could do something with this or other similar tools to automate the CHANGELOG generation based on the commit history. What do you think?

References:
- https://github.com/semantic-release/release-notes-generator

Reply:
Hey @sestrella I like the idea and it seems it will help to generate release notes. Although, I think we will need to strictly follow the semantic release commit format, right?

Reply:
@CristhianMotoche yes, standardizing commit messages would be required to make that work, as most of these tools generate a CHANGELOG based on the git history